week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -16,17 +16,19 @@ function getRemainder(a, b) {
 
 /* ======= TESTS - DO NOT MODIFY ===== */
 
-function test(test_name, expr) {
+const util = require('util');
```

Comment:
This doesn't actually require another file - `util` is built into Node (I was being careful not to require other files). That said, whether we want to avoid use of the magical `require` at all is a good question